src/coercion/date.js

```diff
@@ -0,0 +1,10 @@
+const { isDate } = require('lodash');
+
+module.exports = {
+  type: Date,
+  isCoerced: isDate,
+  default: new Date(null),
```

Comment:
Do you think it's risky to have a mutable object as the default value? I think it would be better to have this object recreated for each instance, what do you think?

Reply:
Following the specs I realized this value is supposed to return always the first unix time when a null value is received, but I agree it can be risky for future equality validations.

Do you think a better option is also accepts a `function` in `default` attribute?